pkg/webhook/policy.go

```diff
@@ -270,6 +270,7 @@ func (h *validationHandler) getValidationMessages(res []*rtypes.Result, req *adm
 		if *logDenies {
 			h.log.WithValues(
 				logging.Process, "admission",
+				logging.Details, r.Metadata,
```

Comment:
```suggestion
				logging.Details, r.Metadata["details"],
```

Should we log `details` field from metadata instead of whole metadata to be consistent with audit?